mini_programs/average.py

```diff
@@ -11,8 +10,5 @@
 avg = sum_of_nums / len(nums)
 
 # Display the numbers & the average
-print('\nYou enter:', end=' ')
-for num in nums:
-    print(num, end=' ')
-
+print('\nYou entered:', *nums, end=' ')
```

Comment:
When calling a function, the `*`operator can be used to unpack an iterable into the arguments in the function call.

i.e: that `print('\nYou entered:', *nums, end=' ')` line is passing all of the items in the nums list into the `print` function call as separate arguments.

Reply:
Thanks for reviewing the code.